app/handlers.js

```diff
@@ -81,7 +81,7 @@ function getCustomFileContent (dependencies) {
     ' * License: www.highcharts.com/license',
     ' */',
     '\'use strict\';',
-    'import Highcharts from \'' + importFolder + 'parts/Globals.js\';',
+    'import Highcharts from \'' + importFolder + 'Core/Globals.js\';',
```

Comment:
This needs some fallback for old branches. Torstein suggests to read the import statement from the product file.

Reply:
With product files the master file of highcharts.src.js is meant.